packages/components/src/components/copy-button/_copy-button.scss

```diff
@@ -107,7 +107,6 @@
     .#{$prefix}--copy-btn__feedback {
       @include tooltip--content('icon');
 
-      z-index: 3;
```

Comment:
can the rest of #7611 (z-index change in `_copy-button.scss`) be reverted as well or does it only apply to this line?

Reply:
I'll test it out and revert it if it can be 👍🏻

Reply:
It seems like we need to keep the `z-index: 10` specific to the multi-line variant. The reason being, if we move it back, the `bx--copy-button` for ALL snippets has a z-index of 10. The assistive text tooltip is a child of the copy button, so it inherits the `z-index: 10` from its parent. Even with the `z-index: 6000` coming from the tooltip z-index, it is still in the global z-index scale of 10, with a sub-index of 6000 inside of the parent context. 
<img width="198" alt="Screen Shot 2021-03-24 at 10 59 02 AM" src="https://user-images.githubusercontent.com/11928039/112360813-f3987900-8c8f-11eb-8b7e-5e486cbe5da6.png">

Hope that makes sense, let me know if you know of any workarounds. My usual way of breaking from the z-index parent stack is to use `transform3d` but for some reason, that doesn't seem to work here.